packages/openassistants/openassistants/contrib/advisor_function.py

```diff
@@ -1,8 +1,8 @@
 import os
 from typing import Literal, Sequence
 
+import httpx
 import pandas as pd
-import requests
```

Comment:
was this the only reference to `requests`?  Can it now be removed as a project dependency?

Reply:
Yup, I introduced requests here. Note, it will probably still be pulled in by some other dep we have, but at least we don't need to require it.

Reply:
ok